src/actor.rs

```diff
@@ -122,7 +123,7 @@ impl<V: ReplicaValidator> Actor<V> {
     /// -----------------------------------------------------------------
 
     /// Step 1. Build a valid cmd for validation of a debit.
-    pub fn transfer(&self, amount: Money, to: AccountId) -> Result<TransferInitiated> {
+    pub fn transfer(&self, amount: Money, to: AccountId) -> Outcome<TransferInitiated> {
         if to == self.id {
             return Err(Error::from("Sender and recipient are the same"));
```

Comment:
Can we use `Outcome::rejected()` here - just to keep all the returns inline.

Reply:
Hm, I trierd it now in the IDE, looks messier IMO?

Reply:
Btw, how do you mean they would all be inline?

Reply:
'In line', typo :) Meaning keeping the returns consistent by using `Outcome`

Reply:
I see though that it is used above..
I think I will fix this up for next PR.